PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -77,27 +77,35 @@
         'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
     )
 )
-MicroEventContentMC = cms.PSet(
-    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
-)
-MicroEventContentMC.outputCommands += [
-        #'keep *_slimmedGenJets*_*_*',
-        'keep *_slimmedGenJets__*',
-        'keep *_slimmedGenJetsFlavourInfos_*_*',
```

Comment:
please re-add ```'keep *_slimmedGenJets__*'``` and ```'keep *_slimmedGenJetsFlavourInfos_*_*'```
to MicroEventContentGEN below in the same way they were here in the ```MicroEventContentMC.outputCommands```

Reply:
ok done. does it matter if before there was
`'keep *_slimmedGenJetsAK8__*',`
and I changed to
`'keep *_slimmedGenJetsAK8_*_*',`
?

Reply:
Please keep it in the same way it was in the baseline master; it matters because now there are some transient products made by slimmedGenJets\*. This will need to be changed in MicroEventContentGEN

I see that you have kept slimmedGenJetsFlavourInfos only in  MicroEventContentMC and not in MicroEventContentGEN. It is perhaps needed in MicroEventContentGEN.